src/components/Options/index.tsx

```diff
@@ -97,47 +110,72 @@ export default class Options extends Component<Props, State> {
   }
 
   render(
-    { class: className, encoderState, preprocessorState, onEncoderOptionsChange }: Props,
+    {
+      sourceImageFile,
+      imageIndex,
+      imageFile,
+      downloadUrl,
+      orientation,
+      encoderState,
+      preprocessorState,
+      onEncoderOptionsChange,
+    }: Props,
     { encoderSupportMap }: State,
   ) {
     // tslint:disable variable-name
     const EncoderOptionComponent = encoderOptionsComponentMap[encoderState.type];
 
     return (
-      <div class={`${style.options}${className ? (' ' + className) : ''}`}>
+      <div class={`${style.options} ${style[orientation]}`}>
```

Comment:
Does `style[…]` prevent any of the minification stuff?

Reply:
Minifiers don't inline `style.whatever` properties at all. It's something I've been meaning to build a solution for in Webpack for a while.